target/linux/ramips/dts/mt7620a_tplink_re200-v1.dts

```diff
@@ -0,0 +1,160 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "tplink,re200-v1", "ralink,mt7620a-soc";
+	model = "TP-Link RE200 v1";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,57600n8";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "re200-v1:green:power";
+			gpios = <&gpio1 0 GPIO_ACTIVE_LOW>;
+		};
+
+		lan {
+			label = "re200-v1:green:lan";
+			gpios = <&gpio2 0 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan {
+			label = "re200-v1:green:wlan";
+			gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0radio";
```

Comment:
If this is the general LED, I would not assign any trigger to it, but assign the appropriate triggers for 2.4 GHz and 5 GHz to the green 2g and 5g LEDs.

Reply:
Yeah, problem is: I couldn't determine the 5G GPIOs. I *assume* they are connected to the mt7610e, but I couldn't verify it.

Reply:
Still, the trigger for 2.4 GHz should be assigned to 2.4 GHz LED, no matter whether it is phy0radio or phy0tpt after all.

Reply:
I changed it to phy1radio, because phy0 is actually the 5G radio (there is no phy1tpt trigger).